frontend/src/core/user/selectors.test.js

```diff
@@ -0,0 +1,44 @@
+import { _isTranslator } from './selectors';
+
+
+describe('isTranslator', () => {
+    it('returns false for non-authenticated users', () => {
+        expect(
+            _isTranslator(
+                { isAuthenticated: false },
+                { locale: 'mylocale', project: 'myproject' }
+            )
+        ).toEqual(false);
+    });
+
+    it('returns true if user is a manager of the locale', () => {
+        expect(
+            _isTranslator(
+                { managerForLocales: ['mylocale'] },
+                { locale: 'mylocale', project: 'myproject' }
+            )
+        ).toEqual(false);
```

Comment:
This test doesn't do what it says it should do. :-) If I understand it correctly, it should have `isAuthenticated: true` in the user part, and assert that the results is true. Btw, you can use `toBeTruthy()` or `toBeFalsy()` here.

Side note, the more I use Jest's assertion API the less I like it. At some point I think I'll introduce [Chai](https://www.chaijs.com/) in Pontoon and switch us to that if you don't mind.

Reply:
Ouch!